src/poisson.jl

```diff
@@ -110,17 +117,38 @@ dΓ = Measure(Γ,degree)
 #
 # With all the ingredients presented so far, we are ready to define the weak form. This is done by defining functions representing the bi-linear and linear forms:
 
+# From strong form to weak form (code viewpoint)
+#
+# In the strong form we have:
+#     -Δu = f in Ω,   u = g on Γ_D,   ∂u/∂n = h on Γ_N.
+#
+# To obtain the weak form, we multiply by a test function v (which vanishes on Γ_D) and integrate by parts, giving:
```

Comment:
I don't think this is necessary. My point of view is that this is an implementation tutorial, not a math class. In the initial header, we state both the strong and weak formulations of the problem and give a reference so that people can have a look at the derivation. If you go to the given reference, you will find an explanation on exactly this, thus making it unnecessary.